packages/backend/src/open_payments/payment/incoming_remote/service.ts

```diff
@@ -0,0 +1,147 @@
+import {
+  AuthenticatedClient,
+  IncomingPayment as OpenPaymentsIncomingPayment,
+  isNonInteractiveGrant
+} from 'open-payments'
+
+import { AccessType, AccessAction, Grant } from '../../grant/model'
+import { GrantService } from '../../grant/service'
+import { BaseService } from '../../../shared/baseService'
+import { Amount, serializeAmount } from '../../amount'
+
+interface CreateRemoteIncomingPaymentArgs {
+  paymentPointerUrl: string
+  description?: string
+  expiresAt?: Date
+  incomingAmount?: Amount
+  externalRef?: string
+}
+
+export interface RemoteIncomingPaymentService {
+  create(
+    args: CreateRemoteIncomingPaymentArgs
+  ): Promise<OpenPaymentsIncomingPayment>
+}
+
+interface ServiceDependencies extends BaseService {
+  grantService: GrantService
+  openPaymentsUrl: string
+  openPaymentsClient: AuthenticatedClient
+}
+
+export async function createRemoteIncomingPaymentService(
```

Comment:
In a separate PR, I want the `ReceiverService` to use this service to fetch remote incoming payments, since it has some similar logic currently. (`ReceiverService` determines whether an incoming payment (or connection) is local and remote, and fetches either)

Reply:
What if the mutation was instead `CreateReceiver`?
I like the idea of making it clear what the purpose of this is.
We could add a `ReceiverService.create` (which could create locally or remotely) but still move remote management to `RemoteIncomingPaymentService` if we wanted.

Reply:
🤔 

- Are you suggesting we also remove `CreateIncomingPayment` and consolidate both remote and incoming payment creation with `CreateReceiver`? 
- If not, is it confusing having both `CreateIncomingPayment` and `CreateReceiver`? Could someone get confused why are they different at first glance?

Reply:
I was thinking we have both
- `CreateIncomingPayment` for a user to get paid
- `CreateReceiver` to pay someone

Reply:
It does fit in well with the quote creation, `receiver` attribute as well.

`RemoteIncomingPayment` GraphQL type can also be also a `Receiver` instead, going to make that change